web/src/invite.ts

```diff
@@ -79,6 +81,10 @@ function get_common_invitation_data(): {
         invite_as,
         stream_ids: JSON.stringify(stream_ids),
         invite_expires_in_minutes: JSON.stringify(expires_in),
+        invitee_emails: pills
+            .items()
+            .map((item) => item.email)
+            .join(","),
```

Comment:
You might as well use the `get_email_from_item` function here if you're going to have one?